server/src/controller/message/message.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, NextFunction } from 'express'
+import messageService from '../../service/message.service'
```

Comment:
```suggestion
import messageService from '@service/message.service'
```

백엔드 설정에도 alias가 등록되어 있으니 사용하면 좋을 것 같습니다